packages/flutter/lib/src/material/about.dart

```diff
@@ -418,6 +430,7 @@ class _LicensePageState extends State<LicensePage> {
     setState(() {
       _loaded = true;
     });
+    Timeline.timeSync('Build scheduled', () { }, flow: Flow.end(flow.id));
```

Comment:
Hmm, it seems what we really want is something like https://api.dartlang.org/stable/1.24.3/dart-html/Performance/mark.html, but strangely it's only available in `dart:html`. Have you filed a feature request?

Reply:
Filed https://github.com/dart-lang/sdk/issues/31950